lib/rucio/core/monitor.py

```diff
@@ -22,14 +22,14 @@
 
 import time
 
-from pystatsd import Client
+from statsd import StatsClient
```

Comment:
pystatsd seems to be not python3 compatible, so I choose another library (https://github.com/jsocol/pystatsd) with almost the same methods